README.md

```diff
@@ -1,90 +1,83 @@
-
-...and more, **without having to rewrite your app!**
-
-* Use your current installer *(no requirement to use MSIX, but there are [reliability/security benefits to using MSIX](https://docs.microsoft.com/windows/msix/overview#key-features)!)*
-* Additive platform APIs *(only add what you need, leave the rest of your app as-is)*
-* Works with Win32, WPF, WinForms, and more apps
-
-Plus, **create beautiful, modern apps for Windows 11 that are backwards compatible** to Windows 10 October 2018 Update (build 17763 / version 1809)!
-
-### Version support
-
-The Windows App SDK aims to work on supported versions of Windows.
-Right now we support down to build 17763 (version 1809/October 2018 Update) of Windows 10.
-Wherever possible, we'll use techniques like '_polyfills_'
-and custom implementations to do the
-heavy lifting for you and make sure your app works across Windows versions and devices.
-In the future, there may be a couple APIs that are dependent on new OS features
-(like new Action Center functionality),
-however we will do our best to ensure
-this is the exception and not the norm, and provide reasonable fallbacks when possible.
-
-
-### Helpful links
-
-* [More details about the Windows App SDK](https://github.com/microsoft/WindowsAppSDK/blob/master/docs/README.md) -
-More on what the project is and how we're approaching the work.
-* [Video on Windows App SDK](https://techcommunity.microsoft.com/t5/video-hub/enterprise-development-futures-project-reunion/m-p/2177278) - Learn about the Windows App SDK in a 22 minute video!
-* [Contributor guide](docs/contributor-guide.md) - How to contribute to the Windows App SDK.
-* [FAQ](docs/faq.md) - Frequently asked questions about the Windows App SDK.
-* [Developer docs](https://aka.ms/windowsappsdkdocs) - Microsoft documentation covering a wide range of Windows App SDK topics.
-* [Samples](https://github.com/microsoft/WindowsAppSDK-Samples) - Discover Windows App SDK samples to assist in learning how to use the latest features!
-
-### Contributing
-
-[![Bugs](https://img.shields.io/github/issues/microsoft/windowsappsdk/bug)](https://github.com/microsoft/WindowsAppSDK/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
-
-_We want to hear from you!_
-
-File a [new issue!](https://github.com/microsoft/WindowsAppSDK/issues/new/choose) Tell us what problem you're
-trying to solve, how you've tried to solve it so far, and what would be the ideal solution for your app. Bonus
-points if there's a gist or existing repo we can look at with you.
+<p align="center">
+  <a href="https://aka.ms/winappsdk">
+    <img src="./docs/images/header.png" alt="Windows App SDK Banner" />
+  </a>
+</p>
+
+<h1 align="center">Windows App SDK</h1>
+
+<h3 align="center">
+  <a href="https://learn.microsoft.com/windows/apps/windows-app-sdk">About Windows App SDK</a>
```

Comment:
```suggestion
  <a href="https://learn.microsoft.com/windows/apps/windows-app-sdk">About</a>
```

Nit: maybe this can be shortened a bit so it's easier to glance over?